shared/util/component-processor/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+import app.futured.kmptemplate.gradle.configuration.ProjectSettings
+
+plugins {
+    id(libs.plugins.kotlin.multiplatform.get().pluginId)
+}
```

Comment:
Does the processor need multiplatform plugin? I think you should be able to get it working using just jvm plugin?